templates/base.html

```diff
@@ -21,7 +21,7 @@
   />
   <link
     rel="stylesheet"
-    href="https://unpkg.com/@creativecommons/vocabulary@v2020.11.3/css/vocabulary.css"
+    href="https://unpkg.com/@creativecommons/vocabulary@2020.11.3/css/vocabulary.css"
```

Comment:
We are deprecating the Vocabulary package in favor of the [`vocabulary-styles` package](https://www.npmjs.com/package/@creativecommons/vocabulary-styles).
```suggestion
    href="https://unpkg.com/@creativecommons/vocabulary-styles@0.x.x/dist/css/vocabulary-styles.css"
```